### PR TITLE
Add CI test using Github Actions

### PR DESCRIPTION
**Pull request by Jialn**
Added build and code checking for playground_test. 

The script installs gazebo dependency every time. To avoid this I also build a docker image https://hub.docker.com/layers/jialn/socialbot/latest/images/sha256-6db01ed73b6516995a4dbf971d4228a53baf1f0c00a91213fea1be468c6defca

But it seems that installing gazebo dependency (need to get some 200~400MB from Azure internal sources) is fast enough, and maybe faster than pulling image from docker-hub (~800MB). The setup and installing progress take about 2.5 minutes.

### CODE REVIEW

File: .github/workflows/python_test.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        mkdir build
+        cd build
+        cmake ..
+        make -j
+        cd ..
+        pip3 install -e .
+    - name: Lint with flake8
+      run: |
+        pip install flake8
+        # distable static check for now
```
Comment (emailweixu) [inline .github/workflows/python_test.yml]: distable => disable

File: .github/workflows/python_test.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python 3.6
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.6
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip3 install -e .
```
Comment (emailweixu) [inline .github/workflows/python_test.yml]: This is a duplicate of line 29.

File: .github/workflows/python_test.yml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        pip3 install absl-py pytest Pillow
```
Comment (emailweixu) [inline .github/workflows/python_test.yml]: should add these dependencies into setup.py instead.

Comment (Jialn) [reply .github/workflows/python_test.yml]: Done

File: .github/workflows/python_test.yml (hunk 1 above)
Comment (Jialn) [reply .github/workflows/python_test.yml]: Fixed

File: .github/workflows/python_test.yml (hunk 2 above)
Comment (Jialn) [reply .github/workflows/python_test.yml]: the first one is needed for cmake and compiling, another one to setup pygazebo after compiling.

File: .github/workflows/python_test.yml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+        make -j
+        cd ..
+        pip3 install -e .
+    - name: Lint with flake8
+      run: |
+        pip install flake8
+        # stop the build if there are Python syntax errors or undefined names
+        flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+        # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
+        flake8 ./python --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
```
Comment (emailweixu) [inline .github/workflows/python_test.yml]: It's better to follow https://github.com/HorizonRobotics/alf/blob/master/.ci-cd/build.sh to use the same style check to be consistent with ALF style.

File: .github/workflows/python_test.yml [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+      run: |
+        pip install flake8
+        # distable static check for now
+        # stop the build if there are Python syntax errors or undefined names
+        # flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+        # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
+        # flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+    - name: Test with pytest
+      run: |
+        GAZEBO_MODEL_PATH=./python/social_bot/models xvfb-run python python/social_bot/envs/play_ground_test.py
```
Comment (witwolf) [inline .github/workflows/python_test.yml]: Is it possible to run all testes by `GAZEBO_MODEL_PATH=./python/social_bot/models xvfb-run python -m unittest discover -p "*_test.py" -v` ?

File: .github/workflows/python_test.yml (hunk 4 above)
Comment (Jialn) [reply .github/workflows/python_test.yml]: Added the the same style check using pre-commit. 
This flake8 checking can be kept, could do static checking for syntax errors or undefined names in corner case that tests does not cover, also it can report warnings like unused import, variables assigned to but never used. max-line-length checking can be ignored since pre-commit already checked that.

File: .github/workflows/python_test.yml [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+        pip3 install pre-commit
+        sudo apt install clang-format-5.0
+        pre-commit install
+        if ! pre-commit run -a ; then
+          git diff
+          exit 0
+        fi
+    - name: Test with pytest
+      run: |
+        cd ./python/social_bot/envs
```
Comment (Jialn) [inline .github/workflows/python_test.yml]: "unittest discover" need to run in this path. Let's only run the tests under env for now.

**Comment by Jialn** [comment]
https://github.com/HorizonRobotics/SocialRobot/issues/102 This issue generates lots of warnings about textures during the tests. I've removed unused texture and stl files in our customized models, and added the original models to our repo because of the texture dependency.
